onadata/apps/api/tests/viewsets/test_organization_profile_viewset.py

```diff
@@ -352,7 +352,7 @@ def test_member_sees_orgs_added_to(self):
         request = self.factory.get('/', **self.extra)
         response = view(request)
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.data, [expected_data])
+        self.assertEqual(dict(response.data[0]), expected_data)
```

Comment:
Why did we have to change this ?

Reply:
yeah for some reason, the response_data returned an OrderedDict object.
I didnt see the correlation with the changes made however.

Reply:
Mhh okay, was the test failing because of it ?

Reply:
lemme confirm this

Reply:
For clarification, this change occurs because as a result of a  sorting issue and does not fail because of the changes to the enketo kwargs